templates/controllers/grid/users/reviewer/readReview.tpl

```diff
@@ -20,35 +20,56 @@
 	<input type="hidden" name="reviewAssignmentId" value="{$reviewAssignment->getId()|escape}" />
 	<input type="hidden" name="submissionId" value="{$reviewAssignment->getSubmissionId()|escape}" />
 	<input type="hidden" name="stageId" value="{$reviewAssignment->getStageId()|escape}" />
-	<p>{translate key="editor.review.readConfirmation"}</p>
+	{if $reviewAssignment->getUnconsidered() == $smarty.const.REVIEW_ASSIGNMENT_UNCONSIDERED}
+		<p>{translate key="editor.review.readConfirmation"}</p>
+	{elseif !$reviewAssignment->getDateCompleted()}
+		<p>{translate key="editor.review.reviewDetails.readConfirmation.description"}</p>
+	{/if}
 	<div id="reviewAssignment-{$reviewAssignment->getId()|escape}">
 		<h2>{$reviewAssignment->getReviewerFullName()|escape}</h2>
-		<span class="pkp_controllers_informationCenter_itemLastEvent">{$reviewAssignment->getDateCompleted()|date_format:$datetimeFormatShort}</span>
 
-		{if $reviewAssignment->getReviewFormId()}
-			{include file="reviewer/review/reviewFormResponse.tpl"}
-		{else}
-			<h3>{translate key="editor.review.reviewerComments"}</h3>
-			{assign var="contents" value=$reviewerComment->getComments()}
-			<span id="reviewContents">
-				{$contents|truncate:250|nl2br|strip_unsafe_html}
-				{if $contents|strlen > 250}<a href="javascript:$.noop();" class="showMore">{translate key="common.more"}</a>{/if}
-			</span>
-			{if $contents|strlen > 250}
-				<span class="hidden">
-					{$contents|nl2br|strip_unsafe_html} <a href="javascript:$.noop();" class="showLess">{translate key="common.less"}</a>
+		{if $reviewAssignment->getDateCompleted()}
+			<p>{translate key="common.completed.date" dateCompleted=$reviewAssignment->getDateCompleted()|date_format:$datetimeFormatShort}</p>
+
+			{if $reviewAssignment->getReviewFormId()}
+				{include file="reviewer/review/reviewFormResponse.tpl"}
+			{elseif $reviewerComment}
+				<h3>{translate key="editor.review.reviewerComments"}</h3>
+				{assign var="contents" value=$reviewerComment->getComments()}
+				<span id="reviewContents">
+					{$contents|truncate:250|nl2br|strip_unsafe_html}
+					{if $contents|strlen > 250}<a href="javascript:$.noop();" class="showMore">{translate key="common.more"}</a>{/if}
```

Comment:
Hi @NateWr, I think I need your help -- this is not trivial at all :-\
Now it looks like this:
```
<div id="reviewContents" style="overflow-y: hidden; height: 70px;"> 
   {$contents|nl2br|strip_unsafe_html}
</div>
{if $contents|strlen > 250}
   <a href="javascript:$.noop();" class="showMore">{translate key="common.more"}</a>
   <a href="javascript:$.noop();" class="showLess" style="display: none">{translate key="common.less"}</a>
{/if}
```
I.e.: 
-- It apparently has to be div (and not a span) element? 
-- The style attribute is just for me to test -- it should be moved somewhere else, I suppose, bu I can leave this to you to decide later?
-- The height should approximately fit (i.e. be smaller than) the 250 strlen contents -- this is how we decide if the links "more" and "less" should be displayed at all. Or do you maybe have a more elegant idea how to solve this?
-- When clicking on "more" and then having "overflow-y: visible", the link "less" is in the middle of the text :-P I.e. we would eventually have to change/toggle the height to "auto" (and back again to "70px" (at the moment)) with Javascript too. What measurement should I then take for the height? Or shall I just ignore the height at the moment and leave the "less" link in the middle of the text?
And here the images, for you to see how it looks like:

![more](https://cloud.githubusercontent.com/assets/378421/12733344/8199bf1c-c93a-11e5-9b88-69f945f73c8e.png)

![less](https://cloud.githubusercontent.com/assets/378421/12733349/877a250c-c93a-11e5-938a-dd6b6f9969ba.png)

THANKS A LOT!!!